firewall/subnet_calculator.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package firewall
+
+import (
+	"fmt"
+	"net"
+)
+
+// SimplifiedSubnet returns IP subnet in x.x.x.x/yy format
+func SimplifiedSubnet(subnet, netmask string) string {
```

Comment:
If we will use `net.IPNet` instead of strings, we will not need to do our own conversion:
https://golang.org/pkg/net/#IPNet.String

Reply:
True, but then we will need to do IPNet construction. I think passing around string is more universal and reduces dependencies. I would just convert to IPNet where its really needed.